Data/Uniques/amulet.lua

```diff
@@ -657,15 +657,18 @@ Star of Wraeclast
 Ruby Amulet
 Source: Vendor recipe
 Variant: Pre 2.6.0
+Variant: Pre 3.8.0
 Variant: Current
 Requires Level 28
+{variant:1,2}Grants level 20 Illusory Warp Skill
+{variant:3}Grants Level 10 Frostblink Skill
 +(20-30)% to Fire Resistance
 (30-50)% increased Cold Damage
 +(10-15)% to all Elemental Resistances
-{variant:1}30% increased Radius of Curse Skills
+{variant:1}30% increased Area of Effect of Curse Skills
 {variant:2}60% increased Area of Effect of Curse Skills
```

Comment:
Need to have the second mod effect versions 2 & 3, like so:
{variant:2,3}60% increased Area of Effect of Curse Skills

Reply:
Fixed, thank you!